qa/suites/rados/cephadm/upgrade/1-start-distro/1-start-centos_8.2-octopus.yaml

```diff
@@ -0,0 +1,39 @@
+os_type: centos
+os_version: "8.2"
```

Comment:
8.3 has been out for a few months now, should we use that as the base?

Reply:
updated